homeassistant/components/amcrest/camera.py

```diff
@@ -132,7 +135,22 @@ async def async_setup_platform(
 
     name = discovery_info[CONF_NAME]
     device = hass.data[DATA_AMCREST][DEVICES][name]
-    async_add_entities([AmcrestCam(name, device, hass.data[DATA_FFMPEG])], True)
+    entity = AmcrestCam(name, device, hass.data[DATA_FFMPEG])
+
+    # 2021.9.0 introduced unique id's for the camera entity, but these were not
+    # unique for different resolution streams.  If any cameras were configured
+    # with this version, update the old entity with the new unique id.
+    serial_number = await hass.async_add_executor_job(lambda: device.api.serial_number)  # type: ignore[no-any-return]
+    registry = entity_registry.async_get(hass)
+    old_entity_id = registry.async_get_entity_id(CAMERA_DOMAIN, DOMAIN, serial_number)
+    if old_entity_id is not None:
+        _LOGGER.info(f"Updating unique id for camera {old_entity_id}")
```

Comment:
```suggestion
        msg  = f"Updating unique id for camera {old_entity_id}"
        _LOGGER.INFO(msg)
```
please do not use f”” in function calls.